src/Symfony/Component/EventDispatcher/Debug/WrappedListener.php

```diff
@@ -118,4 +119,24 @@ public function __invoke(object $event, string $eventName, EventDispatcherInterf
             $this->stoppedPropagation = true;
         }
     }
+
+    private function getNameFromArray(array $listener): ?string
+    {
+        if ($listener[0] instanceof \Closure) {
+            try {
```

Comment:
is the try/catch required?

Reply:
Good catch, I put it to catch `ReflectionException` but it won't because the passed function exists.